server/controllers/testqueries.js

```diff
@@ -0,0 +1,27 @@
+const Promise = require('promise');
+
+let users = [];
+let nextId = 0;
+
+module.exports.reset = () => {
+    users = [];
+    nextId = 0;
+}
+
+module.exports.findOrCreate = (username) => {
+    return new Promise((resolve) => {
+        const user = { username, id: nextId, score: 0 };
+        nextId++;
+        users.push(user);
+        const data = {
+            dataValues: user
+        }
+        resolve([data]);
+    });
+}
+
+module.exports.update = () => {    
```

Comment:
What is this supposed to do?

Reply:
Update the score, but in the testqueries it does nothing for now as there aren't any tests for the leaderboard yet, I'll make them after I create the component